src/components/BDetails/BDetails.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+<details>
+  <summary>
+    <strong>{{summary}}</strong>
```

Comment:
Do we want to use a strong tag here instead of css?

Reply:
We can absolutely use css.